samples/simple/MongoTekmaven_8/Client/Client.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Let's just keep the v9 version of that sample?

Reply:
We could also remove the version related folder to simplify the path a bit as this won't be necessary when the sample is versioned along with the sources

Reply:
I'll remove the v8 version but would like to keep the subfolder so the code doesn't get mixed with the .md file and the related .pngs